### ISSUE TITLE
Cannot set hwclock on UTC on CentOS 7

### ISSUE THREAD

**Issue by lejambon** [saltstack/salt#47031]
### Description of Issue/Question

On CentOS 7, ```modules/timezone.py``` correctly reads whether the hwclock is set to UTC by using timedatectl. However when setting it, it does this...
```
    elif 'RedHat' in __grains__['os_family']:
        __salt__['file.sed'](
            '/etc/sysconfig/clock', '^ZONE=.*', 'ZONE="{0}"'.format(timezone))
```
... which does nothing. This results in the highstate saying this every time but never doing anything...
```
----------
          ID: UTC
    Function: timezone.system
      Result: True
     Comment: Set UTC to True
     Started: 08:31:12.869533
    Duration: 70.262 ms
     Changes:   
              ----------
              utc:
                  True
```

### Setup
State:
```
UTC:
  timezone.system:
    - utc: True
```

### Steps to Reproduce Issue
Take a CentOS 7 server and ensure it has a hwclock set to localtime:
```
# timedatectl set-local-rtc true
```

Apply the above state.

Notice that on every run the state reports changes, however nothing ever does...
```
# timedatectl 
.
 RTC in local TZ: yes
.
```

### Versions Report
```
Salt Version:
           Salt: 2017.7.3
 
Dependency Versions:
           cffi: 1.6.0
       cherrypy: Not Installed
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.7.2
        libgit2: 0.24.6
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: 2.14
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: 0.24.2
         Python: 2.7.5 (default, Aug  4 2017, 00:39:18)
   python-gnupg: Not Installed
         PyYAML: 3.11
          PyZMQ: 15.3.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.1.4
 
System Versions:
           dist: centos 7.4.1708 Core
         locale: ANSI_X3.4-1968
        machine: x86_64
        release: 4.15.15-ck
         system: Linux
        version: CentOS Linux 7.4.1708 Core
```

**Comment by Ch3LL** [comment]
seems we could just change that command to `timedatectl set-timezone`. Seems you have a grasp here want to take a stab at a PR?

**Comment by lejambon** [comment]
Had a stab. A workaround for anyone that needs it is to do:
```
hack-to-set-hwclock-to-utc:
  cmd.run:
    - name: 'timedatectl set-local-rtc false'
    - onlyif: 'timedatectl | grep -i "RTC in local TZ: yes"'

UTC:
  timezone.system:
    - utc: True
```

**Comment by whytewolf** [comment]
ZD-2529

**Comment by poliva83** [comment]
@whytewolf Thanks for linking to zendesk ticket. I didn't know this was already opened.

**Comment by rallytime** [comment]
@garethgreenaway Can you fix this issue for the 2017.7.7 release?

**Comment by garethgreenaway** [comment]
@rallytime Looks like @lejambon opened a PR for this already, just needs to be point it at 2017.7 instead of develop?

**Comment by rallytime** [comment]
@garethgreenaway This will need to be fixed for 2017.7, so if you could help get that across the line/reviewed/backported, that would grand :)
